hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HLESurveyTransformationPipelineBuilder.scala

```diff
@@ -6,19 +6,32 @@ import com.spotify.scio.values.SCollection
 import org.broadinstitute.monster.common.{PipelineBuilder, StorageIO}
 import org.broadinstitute.monster.common.msg._
 import org.broadinstitute.monster.dogaging.jadeschema.table.{HlesDog, HlesOwner}
+import org.slf4j.LoggerFactory
 import upack.Msg
 
 object HLESurveyTransformationPipelineBuilder extends PipelineBuilder[Args] {
 
   implicit val msgCoder: Coder[Msg] = Coder.beam(new UpackMsgCoder)
 
   case class RawRecord(id: Long, fields: Map[String, Array[String]]) {
+    val logger = LoggerFactory.getLogger(getClass)
 
     def getRequired(field: String): String = fields(field).head
 
     def getOptional(field: String): Option[String] = fields.get(field).flatMap(_.headOption)
 
-    def getArray(field: String): Array[String] = fields.get(field).getOrElse(Array.empty)
+    def getBooleanOption(field: String): Option[Boolean] = {
+      val rawFieldValue = this.getOptional(field)
+      if (rawFieldValue.isEmpty) None
+      else if (rawFieldValue.head == "1") Some(true)
+      else if (rawFieldValue.head == "0") Some(false)
```

Comment:
Since it's going back to an `Option` anyways, more conventional would be to `rawFieldValue.map(_ == "1")`